x-pack/plugins/global_search_bar/public/components/search_bar.scss

```
@@ -14,35 +14,20 @@
   border-right: $euiBorderThin; // Adds divider between the tag list and 'Go to' badge
 }
 
-//TODO add these overrides to EUI so that search behaves the same globally (eui/issues/4363)
-.kbnSearchBar {
-  max-width: 100%;
-  will-change: width;
-}
-
 @include euiBreakpoint('xs', 's') {
   .kbnSearchOption__tagsList {
     display: none;
   }
 }
 
-@include euiBreakpoint('m', 'l', 'xl') {
+@include euiBreakpoint('m') {
   .kbnSearchBar {
     width: 400px;
   }
 }
 
 @include euiBreakpoint('l', 'xl') {
```

Sorry to be a stickler here, but the `600px` wide search bar at the large breakpoint is just too large. I'm ok keeping it for very wide screens, but the rest should max at 400px.
```suggestion
@include euiBreakpoint('m', 'l') {
  .kbnSearchBar {
    width: 400px;
  }
}

@include euiBreakpoint('xl') {
```